web/src/components/logs-table.tsx

```diff
@@ -240,6 +250,87 @@ const LogRow: React.FC<LogRowProps> = ({ data, title, showResources }) => {
   return null;
 };
 
+const MetricsLink: React.FC<MetricsLinkProps> = ({ container, logType, namespace, pod }) => {
+  const { t } = useTranslation('plugin__logging-view-plugin');
+
+  const [isLoading, setIsLoading] = React.useState<boolean>(false);
+  const [error, setError] = React.useState<Error | undefined>();
+
+  const checkForMetrics = React.useCallback(() => {
+    setError(undefined);
+    const queryLabels: string[] = [];
+    if (container) {
+      queryLabels.push(`kubernetes_container_name="${container}"`);
+    }
+    if (namespace) {
+      queryLabels.push(`kubernetes_namespace_name="${namespace}"`);
+    }
+    if (pod) {
+      queryLabels.push(`kubernetes_pod_name="${pod}"`);
+    }
+
+    // We require a log type and at least one search label value to get related metrics
+    if (logType && queryLabels.length) {
+      setIsLoading(true);
+
+      const { request } = listGoals({
+        goalsRequest: {
+          start: {
+            class: `log:${logType}`,
+            queries: [`log:${logType}:{${queryLabels.join(',')}}`],
+          },
+          goals: ['metric:metric'],
+        },
+      });
+
+      request()
+        .then((response: Korrel8rResponse) => {
+          setIsLoading(false);
+
+          response.some((goal) => {
+            if (goal?.class === 'metric:metric') {
+              // Use the first goal query that has results and a query
+              const query = goal?.queries?.find((q) => q?.count > 0 && q?.query)?.query;
+
+              if (query) {
+                // Strip korrel8r class off the beginning of the query string
+                const promQL = query.replace(/^metric:metric:/, '');
+
+                const metricsUrl = new URL(window.location.href);
+
+                const params = new URLSearchParams();
+                params.set('query0', promQL);
+                metricsUrl.pathname = '/monitoring/query-browser';
+                metricsUrl.search = params.toString();
+
+                // open the metrics page in a new tab
+                window.open(metricsUrl, '_blank');
```

Comment:
Do we want to open in a new tab? I was thinking that we would redirect in the same tab.

Reply:
Until we have the debugging panel for easy navigation, I think is valuable for users to keep the initial log list and click on result items for metrics that help troubleshoot

Reply:
I agree, I think opening a new tab would be useful for navigating the logging/metrics pages we currently have.